### PR TITLE
fix data backup logic

### PR DESCRIPTION
**Pull request by yjbanov**
/cc @devoncarew @sethladd

### CODE REVIEW

**Comment by sethladd** [comment]
lgtm

**Comment by yjbanov** [comment]
post-LGTM change: encode timestamp in the backup directory

File: bin/build.dart [hunk 1]
```diff
@@ -48,8 +49,11 @@ Future<Null> prepareDataDirectory() async {
   if (!exists(config.backupDirectory))
     mkdirs(config.backupDirectory);
 
-  if (!exists(config.dataDirectory))
-    move(config.dataDirectory, to: config.backupDirectory);
+  if (exists(config.dataDirectory)) {
+    DateFormat dfmt = new DateFormat('yy-MM-dd-HHmmss');
```
Comment (sethladd) [inline bin/build.dart]: Does this format work on the other side?

I know this one does: `2011-08-12T20:17:46.384Z`

**Comment by sethladd** [comment]
lgtm if the date format that you chose is something we're standardizing on (and is easy to decode on the other side).

File: bin/build.dart (hunk 1 above)
Comment (yjbanov) [inline bin/build.dart]: Oh, we're not sending it to Fb. Just a local backup. The format was chosen only to sort well when listing the directory.

Comment (yjbanov) [inline bin/build.dart]: Should probably use `yyyy` though.

Comment (yjbanov) [inline bin/build.dart]: Done.

**Pull request merged**